arelle/PackageManager.py

```diff
@@ -20,8 +20,8 @@
 try:
     from collections import OrderedDict
 except ImportError:
-    OrderedDict = dict # python 3.0 lacks OrderedDict, json file will be in weird order 
-    
+    OrderedDict = dict # python 3.0 lacks OrderedDict, json file will be in weird order
+
```

Comment:
Thanks for the formatting cleanup. For future reference it's easier to review if you make formatting changes as a separate initial commit.

Reply:
Noted, will do!